map.json

```diff
@@ -33,5 +33,10 @@
 		"id" : "win-bcdedit-ignoreAllFailure",
 		"path" : "src/windows/win-ignoreAllFailures.ps1",
 		"description" : "Ignore errors if there is a failed boot, failed shutdown, or failed checkpoint. The computer will attempt to boot normally after an error occurs."
+	},
+	{
+		"id" : "linux-alar-fki",
+		"path" : "src/linux/alar-fki.sh",
+		"description" : "alar-fki allows to recover a failed VM. Three recover scenarios are possible: fstab, initrd and kernel. NOTE: use option --run-on-repair"
```

Comment:
Azure Linux Auto Recover (alar) fixes three scenarios fstab, initrd and kernel (fki) issues. NOTE: only use with the option --run-on-repair after creating a repair vm.

Reply:
without the --run-on-repair the script will not work correct

Reply:
Maybe add a reference to the readme file in the description?

Azure Linux Auto Recover (alar) fixes three scenarios fstab, initrd and kernel (fki) issues. NOTE: only use with the option --run-on-repair after creating a repair vm. Details: https://github.com/Azure/repair-script-library/blob/master/src/linux/common/helpers/alar/README.md